projects/canopy/src/lib/forms/radio/radio-button--segment.component.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../../styles/mixins';
+
+.lg-radio-button--segment {
+  display: flex;
+  flex-grow: 1;
+  flex-basis: 0;
+  align-items: stretch;
+  padding: 0 var(--space-xxxs);
+  margin: var(--space-xxxs) 0;
+
+  + .lg-radio-button--segment {
+    border-left: var(--radio-segment-separator-border-width) solid
+      var(--radio-segment-separator-border-color);
+  }
+
+  .lg-radio-button__label {
+    display: flex;
+    padding: var(--space-xxxs) var(--space-sm);
+    border-radius: var(--border-radius-sm);
+    min-width: var(--radio-segment-label-min-width);
+    width: 100%;
```

Comment:
I tried that before and it wasn't working, the text doesn't wrap automatically when we reach the `min-width`:
![Screenshot 2020-12-04 at 11 54 28](https://user-images.githubusercontent.com/8397116/101160790-851fa480-3627-11eb-965d-3c1844471da2.png)

I also tried to set `flex-basis` instead of `min-width` but that causes issues with the length of the segment when we set `inline-flex`:
![Screenshot 2020-12-04 at 11 58 52](https://user-images.githubusercontent.com/8397116/101161766-383ccd80-3629-11eb-99aa-6720ead97745.png)

I might be missing something ;)

Reply:
Updated to use `flex-grow: 1`.